ui/modal/modalFirstSubscription/view.jsx

```diff
@@ -2,35 +2,64 @@
 import React from 'react';
 import { Modal } from 'modal/modal';
 import Button from 'component/button';
+import * as PAGES from 'constants/pages';
 
 type Props = {
   closeModal: () => void,
+  accessToken: string,
+  user: any,
+  doAuth: () => void,
+  history: { push: string => void },
+  location: UrlLocation,
 };
 
 const ModalFirstSubscription = (props: Props) => {
-  const { closeModal } = props;
+  const {
+    closeModal,
+    accessToken,
+    user,
+    history,
+    location: { pathname },
+  } = props;
 
   return (
     <Modal type="custom" isOpen contentLabel="Subscriptions 101" title={__('Subscriptions 101')}>
       <div className="section__subtitle">
-        <p>{__('You just subscribed to your first channel. Awesome!')}</p>
-        <p>{__('A few quick things to know:')}</p>
-      </div>
-      <ol className="section">
-        <li>
-          {__(
-            'This app will automatically download new free content from channels you are subscribed to. You may configure this in Settings or on the Subscriptions page.'
-          )}
-          {__('(Only available on the desktop app.)')}
-        </li>
-        <li>
-          {__(
-            'If we have your email address, we will send you notifications related to new content. You may configure these emails from the Help page.'
+        <p>{__('Awesome! You just subscribed to your first channel.')}{' '}
+          {/* @if TARGET='web' */}
+          {__('You will receive notifications related to new content.')}
+          {/* @endif */}
+          {/* @if TARGET='app' */}
+          { user && user.primary_email ? (
```

Comment:
For users without an email, say:

"Create a lbry.tv account to receive notifications related to new content" and have the "update email preferences" link be "Create lbry.tv account" and link to `/$/signin`